classes/notification/PKPNotificationOperationManager.inc.php

```diff
@@ -391,7 +397,10 @@ private function sendNotificationEmail($request, $notification) {
 				'siteTitle' => $context?$context->getLocalizedName():$site->getLocalizedTitle()
 			));
 			$mail->addRecipient($user->getEmail(), $user->getFullName());
-			if (!$mail->send()) {
+			if (is_callable($mailConfigurator)) {
+				$mail = $mailConfigurator($mail);
+			}
+			if (!$mail->send() && $request->getUser()) {
```

Comment:
This is another change that I've made because of the command-line/scheduler, to avoid errors.